product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tagging Cloud Service Providers"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to resources created on Your Cloud Account for fine-grained access control and granular cost data at the bucket level.
```

Comment:
```suggestion
Tags are key-value pairs you can apply to the cloud provider resources created on Your Cloud Account for fine-grained access control and granular cost data.
```

Reply:
resources that are tagged are :
- EC2 instances
- S3 buckets
- Loadbalencers
- Volumes